pom.xml

```diff
@@ -76,6 +76,7 @@
                 <module>vaadin-dev-bundle</module>
                 <module>vaadin-prod-bundle/pom-unoptimized.xml</module>
                 <module>vaadin-prod-bundle</module>
+                <module>vaadin-platform-react-hybrid-test</module>
```

Comment:
The test module is behind a profile and added in the standard list of modules, maybe we should excluded it from here and leave it only behind a profile?

Reply:
Yes.